### DISCUSSION TITLE
Klick'r 3.0.2

### DISCUSSION THREAD

**Discussion by Nain57** [Nain57/Smart-AutoClicker#477] [Announcements]
## BugFixes

* Fix a crash on devices without default web browser.
* Fix Pause action animation
* Fix Quick Settings Tile crash upon update on OnePlus8
* Fix a crash when displaying an IntentAction with a ComponentName
* Handle possible preferences file corruption
* Fix centering issue in dumb actions display
* Fix a possible crash on Samsung devices when starting the floating menu

<hr /><em>This discussion was created from the release <a href='https://github.com/Nain57/Smart-AutoClicker/releases/tag/3.0.2'>Klick'r 3.0.2</a>.</em>

**Comment by kimxdhaha** [comment]
The new actions UI is a complete mess.. Hard to insert new action, the order is wrong.. And its freezing after you added all actions and trying to rearrange cause it doesn't follow the order..

**Reply by Nain57** [reply]
Old Action UI was letting people create tons of actions, which is not how Klick'r is designed to be used. This new UI will stay to enforce understanding of new users.

If there is an issue with it, please open a bug a be constructive instead of qualifying a developer's work as "mess"

**Comment by kimxdhaha** [comment]
Sorry I didnt mean that way.. I love this app:( Will try to gather the info report it

**Reply by Nain57** [reply]
No worries, I was in a bad mood, just out of work, sorry for my direct answer. I'm waiting for your feedback!
By the meantime, if you are using 3.0.2, you might to check the "More" tab in the scenario edition dialog ;)

**Comment by kimxdhaha** [comment]
Got it! Thank you for your helping and once again sorry to you and all the developers who work on this project, I didn't mean to disrespectful or something, my English is not good, Is there any way I can buy you a beer or something for supporting your hardworking?

**Comment by Bless-Yan** [comment]
Hello i have a question, is it possible to make a scenario in which theres an action that holds a button and tap on somewhere else in the screen ? This is the only thing i would wish from the app. Quick feedback : i've been using the apps for multiple games and it's great so far so thanks a lot for developping it !

**Comment by VelonWoo** [comment]
If there could be an operation to end and stop, that would be even better!

**Comment by mh3224552** [comment]
Hello everyone,
Can anyone tell me this, I have an action, I have to get it done at 50%, how will it be done Example: I have to perform an action once upon detecting a condition, then if that condition occurs again then I don't have to perform the action again when it occurs for the third time work action, it goes on in a loop like this

**Comment by kimxdhaha** [comment]
Add a trigger for that action to disable itself

**Reply by mh3224552** [reply]
Can you explain little bit

**Comment by kimxdhaha** [comment]
Add a "change event state" in your action to disable the action itself so that so wont detect again until next action turn it back on
